lib/checks/keyboard/has-at-least-one-main.js

```diff
@@ -0,0 +1,3 @@
+var main = document.querySelector('main,div[role="main"]');
```

Comment:
We already have a check for this, see [lib/checks/navigation/landmark.js](https://github.com/dequelabs/axe-core/blob/develop/lib/checks/navigation/landmark.js).